src/components/EventItem/EventItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Image, StyleSheet, Text, TouchableOpacity} from 'react-native';
+import {Constants, fontStyles} from '../../constants';
+import {horizontalScale, verticalScale} from '../../utils';
+import {EventItemProps} from './EventItem.types';
+
+const EventItem = ({name, imageUrl, onPress}: EventItemProps) => {
+  return (
+    <TouchableOpacity onPress={onPress} style={styles.container}>
```

Comment:
Every reusable component should have a style prop for the outer most wrapper

Reply:
Added and noted.